components/search/list-renderer/file.js

```diff
@@ -26,6 +27,17 @@ export default function File({ item: file, accessoryData = null }) {
   ].filter(Boolean);
   const summary = file.summary;
   const fileSet = accessoryData?.[file.file_set];
+  const seqspecOfFileLinks =
+    file.seqspec_of?.length > 0 && accessoryData
+      ? file.seqspec_of
+          .map((key, index) => [
+            <Link key={key} href={key}>
+              {accessoryData[key].accession}
+            </Link>,
+            index < file.seqspec_of.length - 1 && ", ",
+          ])
+          .flat()
+      : [];
```

Comment:
Same thing as the analysis-set.js file, using `<SeparatedList>`.

Reply:
Replaced.